ArduPlane/Parameters.cpp

```diff
@@ -637,35 +637,34 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     GSCALAR(scaling_speed,        "SCALING_SPEED",    SCALING_SPEED),
 
-    // @Param: MIN_GNDSPD_CM
+    // @Param: MIN_GROUNDSPEED
```

Comment:
Should be GROUNDSPEED_MIN - same structure as AIRSPEED_MIN/MAX.

Reply:
I prefer MIN_GROUNDSPEED